pkg/platform/functions/remix-server/regional-server.mjs

```diff
@@ -113,13 +113,16 @@ const createApigHandler = (build) => {
         "Transfer-Encoding": "chunked",
       },
     };
-    if (response.body) {
-      const reader = response.body;
-      const writer = awslambda.HttpResponseStream.from(
+
+    const writer = awslambda.HttpResponseStream.from(
         responseStream,
         httpResponseMetadata,
-      );
-      await streamToNodeStream(reader.getReader(), responseStream);
+    );
+
+    if (response.body) {
+      await streamToNodeStream(response.body.getReader(), responseStream);
+    } else {
+      writer.write(" ");
       writer.end();
     }
```

Comment:
Shouldn't we do a `writer.end()` in all cases?

```suggestion
      writer.write(" ");
    }
      writer.end();
```

Reply:
Yes, but StreamToNodeStream already has write.end(), so both cases already do. I'd be inclined to do away with that function and inline that branch, but left it is as is, assuming @thdxr would do that if he wanted it